pkgs/tools/networking/sstp/default.nix

```
@@ -28,7 +28,6 @@ stdenv.mkDerivation rec {
     description = "SSTP client for Linux";
     homepage = "http://sstp-client.sourceforge.net/";
     platforms = platforms.linux;
```

```suggestion
    platforms = platforms.linux;
    maintainers = with maintainers; [ ];
```

Maybe just keep meta.maintainers an empty list.